app/javascript/src/Stats/lib/csv_link.js

```diff
@@ -26,7 +26,7 @@ export class StatsCSVLink extends StatsUI {
 
   buildCSVString (data) {
     let csvMatrix = this._parseData(data)
-    return csvMatrix.map(cell => cell.join(',')).join('%0A')
+    return csvMatrix.map(row => row.map(cell => encodeURIComponent(cell)).join(',')).join('%0A')
```

Comment:
works locally, what I'm concerned though, how did you choose the `encodeURIComponent` function? This encoding stuff is a little tricky sometimes to get right

Reply:
Well... This is a URL, so I thought that should work :sweat_smile: